src/main/scala/sbtbuildinfo/BuildInfo.scala

```diff
@@ -2,14 +2,77 @@ package sbtbuildinfo
 
 import sbt._, Keys._
 
-case class BuildInfoResult(identifier: String, value: Any, typeExpr: TypeExpression)
+case class BuildInfoResult(identifier: String, value: Any, typeExpr: TypeExpression) {
+  def toStringTuple: (String, String) = (identifier, value.toString)
```

Comment:
I don't know if `value.toString` belongs to this class. sbt-buildinfo has a concept called `BuildInfoRenderer` that takes ` Seq[BuildInfoResult]` and render them, so I feel like putting `toStringTuple` here feels wrong.